src/validation.cpp

```diff
@@ -508,6 +508,19 @@ class MemPoolAccept
             };
         }
 
+        /** Parameters for a single transaction within a package. */
+        static ATMPArgs SingleInPackageAccept(const ATMPArgs& package_args) {
```

Comment:
Ah, good point. I don't have a good reason as to why BIP125 should ever be disallowed here. Will change it to true for `SingleInPackageAccept`

Reply:
If the idea is that everything should be just as if this was a single transaction, why introduce a separate `SingleInPackageAccept` function at all, instead of just creating a `SingleAccept()` ATMPArgs set?

Reply:
I just figured it would be safer to have a constructor converting package-to-single ATMPArgs. There also is a difference (albeit small) - `SingleInPackageAccept` should never have `bypass_limits=true`, so I don't want the caller to have the option to set it.